src/rules/trailingCommaRule.ts

```diff
@@ -86,21 +187,21 @@ class TrailingCommaWalker extends Lint.AbstractWalker<Options> {
                 case ts.SyntaxKind.NamedImports:
                 case ts.SyntaxKind.NamedExports:
                     this.checkList((node as ts.ArrayLiteralExpression | ts.BindingPattern | ts.NamedImportsOrExports).elements,
-                                   node.end);
+                                   node.end, node);
```

Comment:
sure, you need to duplicate some case blocks, but that's ok
they are only grouped that way, because the structure of their AST node is similar. separating them might even make the code easier to understand